### PR TITLE
ARROW-7847: [Website] Add blog post about fuzzing the IPC layer

### PR DESCRIPTION
**Pull request by pitrou**

### CODE REVIEW

File: _posts/2020-04-xx-fuzzing-arrow-ipc.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+Apache Arrow aims to allow fast and seamless data interchange between
+heterogenous runtimes and environments.  Whether using the
+[Flight](https://arrow.apache.org/docs/format/Flight.html) RPC layer,
+the Feather file format, the
```
[the remark this answers is not in the document]
Comment (pitrou) [reply _posts/2020-04-xx-fuzzing-arrow-ipc.md:34]: Done, thank you.

**Comment by pitrou** [comment]
@nealrichardson Do you want to add something?

File: _posts/2020-04-01-fuzzing-arrow-ipc.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+To validate the robustness of the Arrow C++ IPC reader (which also underlies
+the Python, C/GLib, R and Ruby bindings), we
+[successfully submitted](https://github.com/google/oss-fuzz/pull/3233)
+the Arrow project to OSS-Fuzz, a continuous fuzzing initiative for critical
+open source projects, provided by Google.
+
+## What is being fuzzed
+
+As of this writing, the `RecordBatchStreamReader` and `RecordBatchFileReader`
```
[the remark this answers is not in the document]
Comment (wesm) [reply _posts/2020-04-01-fuzzing-arrow-ipc.md:49]: It's mentioned at the end of the doc, but with the caveat that there are still various unfixed issues

[the remark this answers is not in the document]
Comment (pitrou) [reply _posts/2020-04-01-fuzzing-arrow-ipc.md:49]: Yes, I hesitate putting it forward as we haven't fixed all issues yet (also, as you mentioned, not all encodings may be covered currently: someone should check). We can publish another post when we feel it is finally stable on that front.

**Comment by wesm** [comment]
If you use the GitHub UI to merge can you squash?

**Comment by pitrou** [comment]
Oops, sorry. I'll try to next time :-)

**Comment by pitrou** [comment]
Ah, I hadn't realized there's a merge script in this repo as well. Sorry!

**Pull request merged**